earth_enterprise/src/fusion/config/geconfigureassetroot.cpp

```diff
@@ -77,6 +78,8 @@ void usage(const char *prog, const char *msg = 0, ...) {
      "    [--assetroot <dir>]       selected asset root or, modify the local\n"
      "                              path definition for an existing volume,\n"
      "                              or to add a volume definition.\n"
+     "  --listvolumes               List available volumes in the asset root.\n"
```

Comment:
I like adding host as a separate parameter rather than changing this one. @tst-ppenev can you write a follow-up ticket for that change?